lib/vmstatus/list.rb

```diff
@@ -38,26 +39,36 @@ def output(results)
                 :red
               end
 
-      hostname = vm.hostname.length > 15 ? (vm.hostname[0..12] + "...") : vm.hostname
-      on = vm.on? ? 'on' : 'off'
-      running = vm.running? ? '*' : '!'
-
-      checkout = vm.checkout ? vm.checkout.to_s : ' '
-      ttl = vm.ttl ? ("%8.2fh" % vm.ttl) : 'never'
-      user = vm.user ? vm.user : ' '
+      hostname = vm.hostname.length > 24 ? (vm.hostname[0..21] + "...") : vm.hostname
+      vmip = vm.vmip ? vm.vmip : "N/A"
+      dnsip = vm.dnsip ? vm.dnsip: "N/A"
+      if dnsip != vmip
+        dnsip = "*#{dnsip}"
+      end
+      clusterhost = vm.clusterhost ? vm.clusterhost : "N/A"
       type = vm.type ? (vm.type.length > 22 ? (vm.type[0..18] + "...") : vm.type) : 'unknown'
+      # maps the vmpooler host to CI type, redis used to run on the same host as the vmpooler application.
       pooler = case vm.vmpooler
                when 'vmpooler'
                  'ci-old'
                when 'vmpooler-cinext'
                  'ci-next'
                when 'vmpooler-dev'
                  'ci-dev'
+               when 'vmpooler-redis-prod-2.delivery.puppetlabs.net'
+                 'ci-next'
```

Comment:
Maybe a task for later, but I assume this needs updating for `vmpooler-prod` and `vmpooler-stage` in k8s too.

Reply:
yes we should. Its for statsd though like what the prefix we should use. So not that useful if we are removing statsd